server/src/internalClusterTest/java/org/opensearch/search/SearchCancellationIT.java

```diff
@@ -241,7 +241,8 @@ public void testCancellationDuringQueryPhaseUsingRequestParameter() throws Excep
             .execute();
         awaitForBlock(plugins);
         // sleep for cancellation timeout to ensure scheduled cancellation task is actually executed
```

Comment:
This comment is no longer accurate. How about a static `sleepForTimeout()` or similar method so you only have to write the comment once?

Also, any idea why 2 seconds is used everywhere here? Would be great to make that something shorter (like 100ms) to speed up the test.

Reply:
But I was having so much fun copying and pasting with _The Key_.   Sure, I'll make a static call.  Not sure what the best minimum should be, but I'm guessing I could make it 1 second at least?

Reply:
Sure. You'll make it twice as fast!